source/modes/GameMode.cpp

```diff
@@ -450,6 +458,7 @@ bool GameMode::mousePressed(const OIS::MouseEvent& arg, OIS::MouseButtonID id)
 
     if (id == OIS::MB_Middle)
     {
+        inputManager.mDragType=InputManager::rotateAxisXY;
```

Comment:
Since you used the middle clic for rotation, it should be a good thing to display the status window in the released event. Note that it is not blocking and if you don't feel like doing it, I can handle this part. Just tell me.